tests/examplefiles/blazor.razor

```diff
@@ -0,0 +1,20 @@
+<div class="card" style="width:22rem">
```

Comment:
This file should be named `tests/examplefile/blazor/blazor.razor` (inside a subdirectory) and you need to commit the test output obtained with `pytest --update-goldens tests/examplefiles/blazor`.